src/blocks/Login/Login.tsx

```diff
@@ -0,0 +1,52 @@
+import Ublock from '@src/components/Ublock/Ublock'
+import React from 'react'
+
+interface LoginProps {
+    title: string
+    onSubmit?: React.FormEventHandler
+}
+
+const Login: React.FC<LoginProps> = ({ title, onSubmit }): JSX.Element => {
+    const loginForm = (
+        <div className='b-login'>
+            <h2>{title}</h2>
+            <form
+                className='c-form'
+                name='loginform'
+                acceptCharset='UTF-8'
+                onSubmit={onSubmit}
+            >
+                <label htmlFor='user_login'>Username</label>
+                <input
+                    type='text'
+                    id='user_login'
+                    name='log'
+                    className='login__field'
+                    aria-required='true'
+                    autoComplete='off'
+                />
+                <label htmlFor='user_pass'>
+                    Password
+                    <a
+                        className='login__link'
+                        href='https://myone.carleton.ca/'
```

Comment:
Could you, please, add a function to handle the this? I know right now this is the only place you can reset your password but we are trying to make the components agnostic :)